src/components/DateSelector/DateSelector.tsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import './DateSelector.scss';
+import SoftKey from '../SoftKey/SoftKey';
+import TriColListView from '../../views/TriColumnListView/TriColumnListView';
+import BodyTextListItem from '../BodyTextListItem/BodyTextListItem';
+import { getDaysInMonth } from '../../utils/dates';
+
+interface LocalProps {
+  onOK: (res?: Date|null) => void;
+  onCancel: () => void;
+  header: React.ElementType | string | undefined;
+  inputOptions: any;
+  minDate: Date;
+  maxDate: Date;
+  initialDate: Date;
+}
+
+const Months = {
```

Comment:
Should we `Object.freeze` this since it's an enum?

Reply:
Good call, missed this.